torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -96,6 +96,9 @@ void IrPrinter::handle(const IterDomain* id) {
   os_ << "}";
   if (id->isRFactorProduct())
     os_ << "rf";
+  if (id->hasPaddingToMultipleOfWarp()) {
+    os_ << "(padded)";
```

Comment:
can we keep this concise? Something simple like `p` would probably be fine here.

Reply:
Updated thanks.